docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2019-10-16
+ - Added response code after each URL reported on the command line
```

Comment:
"in the output" or "on standard out"?

Reply:
yeah, wasnt completely happy with that - now changed :)